requirements.txt

```diff
@@ -7,5 +7,6 @@ pycparser==2.21
 pyelftools==0.29
 Pygments==2.13.0
 ROPGadget==7.1
+tabulate==0.9.0
```

Comment:
Don't know if it's OK to add this dependency. I'm tired of wasting time formatting output, I figure we should just pick some library and use it throughout the code for consistency.